packages/atomic/src/components/facets-v1/facet-value-link/facet-value-link.tsx

```diff
@@ -1,14 +1,36 @@
 import {FunctionalComponent, h} from '@stencil/core';
+import {FacetValueProps} from '../facet-common';
 
-interface FacetValueLinkProps {
-  value: string;
-}
+export const FacetValueLink: FunctionalComponent<FacetValueProps> = (props) => {
+  const count = props.numberOfResults.toLocaleString(props.i18n.language);
+  const ariaLabel = props.i18n.t('facetValue', {
+    value: props.displayValue,
+    count: props.numberOfResults,
+  });
 
-// TODO: add interactivity, style & part
-export const FacetValueLink: FunctionalComponent<FacetValueLinkProps> = (
-  props
-) => (
-  <div>
-    <a href="#">{props.value}</a>
-  </div>
-);
+  return (
+    <li key={props.displayValue}>
+      <button
```

Comment:
Similar comment as above on the impacts on accessibility of using a `button` element,